cmake/configs/default.cmake

```diff
@@ -51,6 +51,9 @@ hunter_config(CppNetlibUri VERSION 1.0.4-hunter)
 hunter_config(CsvParserCPlusPlus VERSION 1.0.1)
 hunter_config(Eigen VERSION 3.2.4-p0)
 hunter_config(Expat VERSION 2.1.1)
+if(MSVC)
+  hunter_config(getopt VERSION 1.0.0-p0)
```

Comment:
> as there is nothing about it that makes it windows only

Well, except that this package works only on Windows, right? :)

> And the header are the same getopt.h

Okay, this is strong argument. If you want you can keep the name of the package, up to you.